datalad/interface/__init__.py

```diff
@@ -64,7 +64,7 @@
     ])
 
 _group_2metadata = (
-    'Commands for metadata handling',
+    'Metadata',
```

Comment:
minor but might want to avoid modifying this to avoid conflict with https://github.com/datalad/datalad/pull/7014